libs/wingsdk/src/core/app.ts

```diff
@@ -188,6 +188,11 @@ export abstract class CdktfApp extends App {
 
     super(cdktfStack, "Default");
 
+    // TODO: allow the user to specify custom backends
```

Comment:
Is this something on our issues radar? I dont see anything myself.

Reply:
Makes sense to me https://github.com/winglang/wing/issues/2003